packages/lib-classifier/src/plugins/tasks/survey/components/components/Chooser/components/Choices/components/ChoiceButton.js

```diff
@@ -52,7 +52,10 @@ function ChoiceButton({
   return (
     <Button
       ref={choiceButton}
+      aria-checked={selected}
```

Comment:
```suggestion
      aria-checked={selected ? 'true' : 'false'}
```
ARIA states are always strings. There aren't any boolean attributes in ARIA.